internal/operands/metrics/resources.go

```diff
@@ -20,13 +20,47 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 			},
 		},
 		Spec: promv1.PrometheusRuleSpec{
-			Groups: []promv1.RuleGroup{{
-				Name: "cnv.rules",
-				Rules: []promv1.Rule{{
-					Expr:   intstr.FromString("sum(kubevirt_vmi_phase_count{phase=\"running\"}) by (node,os,workload,flavor)"),
-					Record: "cnv:vmi_status_running:count",
-				}},
-			}},
+			Groups: []promv1.RuleGroup{
+				{
+					Name: "cnv.rules",
+					Rules: []promv1.Rule{
+						{
+							Expr:   intstr.FromString("sum(kubevirt_vmi_phase_count{phase=\"running\"}) by (node,os,workload,flavor)"),
+							Record: "cnv:vmi_status_running:count",
+						},
+						{
+							Record: "num_of_running_ssp_operators",
+							Expr:   intstr.FromString("sum(up{pod=~'ssp-operator.*'}) OR on() vector(0)"),
+						},
+						{
+							Record: "num_of_running_template_validator_pods",
+							Expr:   intstr.FromString("sum(up{pod=~'virt-template-validator.*'}) OR on() vector(0)"),
+						},
+						{
+							Alert: "SSPDown",
+							Expr:  intstr.FromString("num_of_running_ssp_operators == 0"),
+							For:   "5m",
+							Annotations: map[string]string{
+								"summary": "All SSP operator pods are down.",
+							},
+							Labels: map[string]string{
+								"severity": "Critical",
```

Comment:
Where are the `severity` levels defined? Maybe `Critical` is too big severity?

For me, critical severity means: "Wake up the cluster admin in the middle of the night to fix this error." Maybe that is too much for missing SSP operator :).

Reply:
We discussed this offline. You can ignore this comment.